pkg/stream/server_frame.go

```diff
@@ -478,6 +482,29 @@ func (c *Client) metadataUpdateFrameHandler(buffer *bufio.Reader) {
 	}
 }
 
+func (c *Client) streamStatusFrameHandler(readProtocol *ReaderProtocol,
+	r *bufio.Reader) {
+
+	c.handleGenericResponse(readProtocol, r)
+
+	count, _ := readUInt(r)
+	streamStatus := make(map[string]int64)
```

Comment:
Neat pick: since you know the size of the map (because you have `count`), you can pass `count` to `make` and create the map with that capacity, so that it doesn't grow your map more than necessary.

Reply:
yes, but we are speaking about 3 elements